5.0/fr/0x91-Appendix-B_References.md

```diff
@@ -0,0 +1,43 @@
+# Annexe B : Références
+
+Les projets OWASP suivants sont les plus susceptibles d’être utiles aux utilisateurs/adoptants de cette norme :
+
+## Projets principaux de l'OWASP
+
+1. OWASP Top 10 Project : [https://owasp.org/www-project-top-ten/](https://owasp.org/www-project-top-ten/)
+2. OWASP Web Security Testing Guide : [https://owasp.org/www-project-web-security-testing-guide/](https://owasp.org/www-project-web-security-testing-guide/)
+3. OWASP Proactive Controls : [https://owasp.org/www-project-proactive-controls/](https://owasp.org/www-project-proactive-controls/)
+4. OWASP Software Assurance Maturity Model (SAMM) : [https://owasp.org/www-project-samm/](https://owasp.org/www-project-samm/)
+5. OWASP Secure Headers Project : [https://owasp.org/www-project-secure-headers/](https://owasp.org/www-project-secure-headers/)
+
+## Projet de la série de fiches de triche de l'OWASP
```

Comment:
"triche" is not great in French. Align with whatever we chose in the previous chapters.